gamebook/game_manager.py

```diff
@@ -0,0 +1,43 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
```

Comment:
This should take a config later on (add an issue?), that looks something like:
```
{
io_type: ['Terminal', 'Web', 'Server']
first_scene: Scene
scenes_dir: String
}
```

Reply:
Resolved? https://github.com/arielofer/gamebook.py/issues/13
@arielofer

Reply:
how do i do that? I'm not sure i understood your example because I'm getting a syntax error
![Screenshot from 2020-05-10 15-19-54](https://user-images.githubusercontent.com/60113369/81499097-b45c7600-92d1-11ea-97ad-c4829ef5cf26.png)

Reply:
@agamm

Reply:
Don't do it right now, I only wanted to verify that #13 is an issue that corresponds to this comment.